src/base/iana/exterr.rs

```diff
@@ -149,6 +149,9 @@ int_enum! {
     /// is otherwise configured to support. Examples of this include
     /// its most recent zone being too old or having expired.
     (INVALID_DATA => 24, b"Invalid Data")
+
+    /// The requested resource record should not appear in query.
```

Comment:
Just a couple of nits about the comment:

- Replace "in query" with "in a query" or "in queries".

- The RFC refers to "any RR _type_" while the comment refers to "resource record" (without type). I think "resource record type" would be more correct.

Reply:
They definitely add clarity, I've made the changes now